### PR TITLE
Fix termination condition deprecation

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: src/solve.jl [hunk 1]
```diff
@@ -15,7 +15,7 @@ end
 
 function DiffEqBase.__solve(prob::DiffEqBase.AbstractSteadyStateProblem, alg::DynamicSS,
         args...; abstol = 1e-8, reltol = 1e-6, odesolve_kwargs = (;),
-        save_idxs = nothing, termination_condition = SteadyStateDiffEqTerminationMode(),
+        save_idxs = nothing, termination_condition = NormTerminationMode(),
```
Comment (ChrisRackauckas) [inline src/solve.jl:18]:
```suggestion
        save_idxs = nothing, termination_condition = NormTerminationMode(DiffEqBase.ODE_DEFAULT_NORM),
```

**Comment by avik-pal** [comment]
Needs to be removed from https://github.com/SciML/SteadyStateDiffEq.jl/blob/820157f2249b000a96d0b49cd792eee13908bdab/test/core.jl#L78

File: test/core.jl [hunk 2]
```diff
@@ -76,7 +76,6 @@ sol2 = solve(prob, DynamicSS(Tsit5()); abstol = 1e-4)
 @test typeof(u0) == typeof(sol2.u)
 
 for termination_condition in [
-    SteadyStateDiffEqTerminationMode(), SimpleNonlinearSolveTerminationMode(),
     NormTerminationMode(), RelTerminationMode(), RelNormTerminationMode(),
```
Comment (ChrisRackauckas) [inline test/core.jl:79]:
```suggestion
    NormTerminationMode(SteadyStateDiffEq.infnorm), RelTerminationMode(), RelNormTerminationMode(),
```

File: test/core.jl [hunk 3]
```diff
@@ -76,10 +76,9 @@ sol2 = solve(prob, DynamicSS(Tsit5()); abstol = 1e-4)
 @test typeof(u0) == typeof(sol2.u)
 
 for termination_condition in [
-    SteadyStateDiffEqTerminationMode(), SimpleNonlinearSolveTerminationMode(),
-    NormTerminationMode(), RelTerminationMode(), RelNormTerminationMode(),
-    AbsTerminationMode(), AbsNormTerminationMode(), RelSafeTerminationMode(),
-    AbsSafeTerminationMode(), RelSafeBestTerminationMode(), AbsSafeBestTerminationMode()
+    NormTerminationMode(SteadyStateDiffEq.infnorm), RelTerminationMode(), RelNormTerminationMode(SteadyStateDiffEq.infnorm),
+    AbsTerminationMode(), AbsNormTerminationMode(SteadyStateDiffEq.infnorm), RelSafeTerminationMode(SteadyStateDiffEq.infnorm),
+    AbsSafeTerminationMode(SteadyStateDiffEq.infnorm), RelSafeBestTerminationMode(), AbsSafeBestTerminationMode()
```
Comment (ChrisRackauckas) [inline test/core.jl:81]:
```suggestion
    AbsSafeTerminationMode(SteadyStateDiffEq.infnorm), RelSafeBestTerminationMode(SteadyStateDiffEq.infnorm), AbsSafeBestTerminationMode(SteadyStateDiffEq.infnorm)
```

**Pull request merged**